bundle/manifests/siteconfig.clusterserviceversion.yaml

```diff
@@ -37,7 +37,7 @@ metadata:
               "tang": [
                 {
                   "thumbprint": "1234567890",
-                  "url": "http://10.0.0.1:7500"
+                  "url": "http://1.0.0.1:7500"
```

Comment:
Why change these? And in any case I'd expect these examples to pull from the reserved subnets for documentation. https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
I meant to change to loopback address! Updated.